__tests__/app.test.js

```diff
@@ -190,3 +190,33 @@ describe('tests for get /api/articles', () => {
     })
 }) 
 
+describe('tests for delete /api/comments/:comment_id', () => {
+    test('returns status 204 and deletes the comment from the database', async () => {
+        const response = await request(app).delete('/api/comments/2').expect(204);
+        const {body} = response
+        expect(body).toEqual({})
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then((result) => {
+            const {comments} = result.body
+            expect(comments.length).toBe(10) //down from 11
+        })
```

Comment:
While this works, the mixed syntax of dealing with promises isn't consistent.
I'd refactor to stick with the .then syntax you've been doing using throughout your app.

Reply:
ah thanks Ali. I usually prefer the one I didn't use here - but had to use async await here to avoid nested thens (hence the mix!)